recipes/cuequivariance-torch/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "cuequivariance-torch" %}
+{% set version = "0.1.0" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NVIDIA/cuEquivariance/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 7921940729ea444fa3688907b68056cfb371aba7c47fde231fd4693f19eee5e7
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install ./cuequivariance_torch -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - hatchling
+    - pip
+  run:
+    - python >={{ python_min }}
+    - cuequivariance
+
+test:
+  imports:
+    - cuequivariance_torch
+  commands:
+    - pip check
+  requires:
+    - pip
+    - python {{ python_min }}
+    - pytorch
```

Comment:
Move this to `run` requirements?

Reply:
I don't know, grayskull put it there and all testing is happy with it.

Reply:
`torch` is not a test only dependency. It is a run dependency. So you need to move it to Line 26. grayskull is confused because they didn't add `torch` as a dependency in pyproject.toml.

Reply:
OK, done